packages/block-library/package.json

```diff
@@ -40,6 +40,7 @@
 		"@wordpress/date": "file:../date",
 		"@wordpress/deprecated": "file:../deprecated",
 		"@wordpress/dom": "file:../dom",
+		"@wordpress/edit-post": "file:../edit-post",
```

Comment:
Let's avoid this cyclic dependency for `@wordpress/block-library` package and leave it as a string for now.